SU2_CFD/include/output_structure.hpp

```diff
@@ -47,7 +47,7 @@
 #include <time.h>
 #include <fstream>
 
-#include "solver_structure.hpp"
+//#include "solver_structure.hpp"
```

Comment:
I was experimenting some stuff. In truth this header file does not need to be included. For the moment, I have included again.